VirtoCommerce.Platform.Web/Web.config

```diff
@@ -122,6 +122,8 @@
         <!--<add key="GoogleTagManager:ContainerId" value="SECRET" />-->
         <!--Use this setting to specify redis channel name to be used by cache. Only instances with the same channel name will recieve messages. -->
         <!--<add key="VirtoCommerce:Cache:Redis:ChannelName" value="(Place name of desired channel here)" />-->
+        <!-- Copy satellite localization assemblies from module bin folder with language code names: Example "de-DE,ru-RU"-->
+        <add key="VirtoCommerce:Localization:ServerLocalizationCodes" value=""/>
```

Comment:
Let's name this setting `VirtoCommerce:Resources:CultureNames`

Reply:
done